### PR TITLE
Move openrasta to new csproj

### PR DESCRIPTION
**Pull request by jchannon**
Hi there!

Thank you so much for contributing this pull request. We love when the community
steps up and share, and we're enormously thankful.

You don't have to do all of those following things, they are what we try and get 
done for code coming in:

 - [x] the code
 - [ ] If it's a non-trivial piece of code, signing a CLA
 - [ ] If it breaks, change, fix or add to existing behaviour, updating
       CHANGELOG.md
 - [ ] If it's a bug fix, tests in the Tests `project`, or scenarios in the `TestRig`.
 - [ ] On top of HEAD, unless it's a bugfix on a previous version
 - [ ] VERSION file updated if not creating a pull-request on top of `master`

If those are needed, and you haven't and can't, we thank you enromously for your
contirbution, and we'll add those before merging the pull request.

Thanks!

The OpenRasta community

### CODE REVIEW

**Comment by serialseb** [comment]
No need for the CLA in this case :)

File: appveyor.yml [hunk 1]
```diff
@@ -48,6 +49,14 @@ nuget:
   project_feed: true
   disable_publish_on_pr: true
 
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  version: '{version}'
+  assembly_version: '{version}'
```
Comment (serialseb) [inline appveyor.yml]: here

**Comment by jchannon** [comment]
Is there anything here you think needs adding etc?

**Comment by serialseb** [comment]
As this still targets .net 46, nope we're good to go, i'm merging it in. Then it's "just" a matter of adding netstandard2. :-D

**Pull request merged**